e2e/_suites/ingest-manager/fleet.go

```diff
@@ -490,22 +879,10 @@ func checkFleetConfiguration() error {
 // createFleetConfiguration sends a POST request to Fleet forcing the
 // recreation of the configuration
 func createFleetConfiguration() error {
-	type payload struct {
-		ForceRecreate bool `json:"forceRecreate"`
-	}
-
-	data := payload{
-		ForceRecreate: true,
-	}
-	payloadBytes, err := json.Marshal(data)
-	if err != nil {
-		log.Error("Could not serialise payload")
-		return err
-	}
-
 	postReq := createDefaultHTTPRequest(fleetSetupURL)
-
-	postReq.Payload = payloadBytes
+	postReq.Payload = `{
+		"forceRecreate": true
+	}`
```

Comment:
Simplification of the payload we added for the CURL code: because we need a string we do not have to create a struct and then serialise it to JSON